content/guides/models/track/log/distributed-training.md

```diff
@@ -95,22 +104,21 @@ The preceding image demonstrates the W&B App UI dashboard. On the sidebar we see
 
 ### Track all processes to a single run
 
-
 {{% alert color="secondary"  %}}
 Parameters prefixed by `x_` (such as `x_label`) are in public preview. Create a GitHub issue at [https://github.com/wandb/wandb](https://github.com/wandb/wandb) to provide feedback.
 {{% /alert %}}
 
 {{% alert %}}
-To track multiple processes to a single run, you must have W&B Python SDK version `>=0.19.0`.
+To track multiple processes to a single run, you must have W&B Python SDK version `>=0.19.5`.
 {{% /alert  %}}
 
 In this approach you use a primary node and one or more worker nodes. Within the primary node you initialize a W&B run. For each worker node, initialize a run using the run ID used by the primary node. During training each worker node logs to the same run ID as the primary node. W&B aggregates metrics from all nodes and displays them in the W&B App UI.
 
 Within the primary node, initialize a W&B run with [`wandb.init`]({{< relref "/ref/python/init.md" >}}). Pass in a `wandb.Settings` object to the `settings` parameter (`wandb.init(settings=wandb.Settings()`) wit with the following:
 
 1. The `mode` parameter set to `"shared"` to enable shared mode.
-2. A unique label for `x_label`. You use the value you specify for `x_label` to identify which node the data is coming from in logs and system metrics in the W&B App UI. If left unspecified, W&B creates a label for you using the hostname and a random hash.
-3. Set the `x_primary` parameter to `True` to indicate that this is the primary node.
+2. A unique label for [`x_label`](https://github.com/wandb/wandb/blob/6cf0e1d5e1da665dd0ce4691ae77fba85a56c5c9/wandb/sdk/wandb_settings.py#L638). You use the value you specify for `x_label` to identify which node the data is coming from in logs and system metrics in the W&B App UI. If left unspecified, W&B creates a label for you using the hostname and a random hash.
```

Comment:
Try to avoid linking to a SHA. Can you just link to `main`? Also a few lines down.

Reply:
ooops. Good catch